File: apps/problems/Find-Median/full-boilerplate/function.java

    import java.io.*;
    import java.util.*;
    
    public class Main {
        
        ##USER_CODE_HERE##
    
        public static void main(String[] args) {
            String filePath = "/Users/vishal/Desktop/PROJECTS/algoearth3/apps/problems/find-median/tests/inputs/##INPUT_FILE_INDEX##.txt"; 
            List<String> lines = readLinesFromFile(filePath);
            int size_numbers = Integer.parseInt(lines.get(0).trim());

            List<Integer> numbers = new ArrayList<>(size_numbers);

            String[] inputStream = lines.get(1).trim().split("\s+");

            for (String inputChar : inputStream)  {

              numbers.add(Integer.parseInt(inputChar));

            }

            float result = findMedian(numbers);
            System.out.println(result);
        }
        public static List<String> readLinesFromFile(String filePath) {
            List<String> lines = new ArrayList<>();
            try (BufferedReader br = new BufferedReader(new FileReader(filePath))) {
                String line;
                while ((line = br.readLine()) != null) {
                    lines.add(line);
                }
            } catch (IOException e) {
                e.printStackTrace();
            }
            return lines;
        }
    }

File: apps/problems/Sort-Array/boilerplate-full/function.js
##USER_CODE_HERE##
    
    const input = require('fs').readFileSync('/Users/vishal/Desktop/PROJECTS/algoearth3/apps/problems/sort-array/tests/inputs/##INPUT_FILE_INDEX##.txt', 'utf8').trim().split('\n').join(' ').split(' ');
    const size_values = parseInt(input.shift());
const values = input.splice(0, size_values).map(Number);
    const result = sortArray(values);
    console.log(result);
        